scripts/benchmarking/train_gaussian_denoiser.py

```diff
@@ -0,0 +1,247 @@
+        batch_normalisation=True,
+    )
+    model = DnCNN(model_params).cuda()
+    config = {
+        "depth": params["depth"],
+        "int_channels": params["channels"],
+        "kernel_size": params["kernel_size"],
+        "bias_free": params["bias_free"],
+        "enforce_positivity": params["enforce_positivity"],
+        "lr": params["lr"],
+        "epochs": params["epochs"],
+        "noise_level": params["noise_level"],
+        "commit_hash": commit_hash,
+    }
+elif params["model"] == "drunet":
+    model_params = LIONParameter(
+        in_channels=1,
+        out_channels=1,
+        int_channels=params["channels"],
+        kernel_size=(params["kernel_size"], params["kernel_size"]),
+        n_blocks=params["n_blocks"],
+        use_noise_level=False,
+        bias_free=params["bias_free"],
+        act="leaky_relu",
+        enforce_positivity=params["enforce_positivity"],
+    )
+    model = DRUNet(model_params).cuda()
+    config = {
+        "n_blocks": params["n_blocks"],
+        "int_channels": params["channels"],
+        "kernel_size": params["kernel_size"],
+        "bias_free": params["bias_free"],
+        "enforce_positivity": params["enforce_positivity"],
+        "lr": params["lr"],
+        "epochs": params["epochs"],
+        "noise_level": params["noise_level"],
+        "commit_hash": commit_hash,
+    }
+elif params["model"] == "gs_drunet":
+    model_params = LIONParameter(
+        in_channels=1,
+        out_channels=1,
+        int_channels=params["channels"],
+        kernel_size=(params["kernel_size"], params["kernel_size"]),
+        n_blocks=params["n_blocks"],
+        use_noise_level=False,
+        bias_free=params["bias_free"],
+        act="elu",
+        enforce_positivity=params["enforce_positivity"],
+    )
+    model = GSDRUNet(model_params).cuda()
+    config = {
+        "n_blocks": params["n_blocks"],
+        "int_channels": params["channels"],
+        "kernel_size": params["kernel_size"],
+        "bias_free": params["bias_free"],
+        "lr": params["lr"],
+        "epochs": params["epochs"],
+        "noise_level": params["noise_level"],
+        "commit_hash": commit_hash,
+    }
+else:
+    raise NotImplementedError(f"Model {params['model']} has not been implemented!")
+
+experiment_id = uuid.uuid1()
+experiment_name = f"{params['model']}"
+if params["bias_free"]:
+    experiment_name += "_bias_free"
+if "dncnn" in params["model"]:
+    experiment_name += f"_depth={params['depth']}"
+else:
+    experiment_name += f"_n_blocks={params['n_blocks']}"
+experiment_name += f"_noise_level={params['noise_level']}_{experiment_id}"
+print(experiment_name)
+print(config)
+wandb.init(project="benchmarking_ct", config=config, name=experiment_name)
+
+optimiser = torch.optim.Adam(model.parameters(), lr=params["lr"], betas=(0.9, 0.9))
+random_crop = RandomCrop((256, 256))
+random_erasing = RandomErasing()
+experiment = ct_benchmarking.GroundTruthCT()
```

Comment:
You may have missed to add some files?

Reply:
No, I'm not sure why you think so? `RandomCrop` and `RandomErasing` are from `kornia.augmentation` and imported on line 16.

Reply:
I think you introduced `ct_benchmarking.GroundTruthCT`, its not in main.

Reply:
Yes it is, it was in this pull request https://github.com/CambridgeCIA/LION/pull/93

Reply:
Man, I literally went to the file and contr+f it and could not find it. Not my brightest day.